### PR TITLE
Resolve Issue #320 (Doc config options)

### PR DESCRIPTION
**Pull request by james-d-mitchell**
See Issue #320.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,12 +85,27 @@ If you installed GAP on several architectures, then you must execute the
...
 
-Digraphs vendors `bliss` and `planarity` libraries in the `extern` directory.
+### Configuration options
+
+In addition to the usual autoconf generated configuration flags, the following
+flags are provided.
+
+Option | Meaning
+-------- | ------- 
+--enable-code-coverage        | enable code coverage support
```
Comment (wilfwilson) [inline README.md:100]: This doesn't look very good when read as plain text. Perhaps make it line up if reading in a fixed-width-font?
```suggestion
 Option                       | Meaning
----------------------------- | ---------------------------- 
--enable-code-coverage        | enable code coverage support
```

**Review by wilfwilson — APPROVED** [review]

File: doc/z-chap1.xml [hunk 2]
```diff
@@ -94,6 +94,45 @@ make]]></Listing>
...
+<Table Align="|l|l|">
+ <Caption>Configuration flags</Caption>
+ <HorLine/>
+  <Row>
+   <Item>Option</Item><Item>Meaning</Item>
+  </Row>
+ <HorLine/>
+ <HorLine/>
+ <Row>
+<Item>--enable-code-coverage        </Item><Item> enable code coverage support</Item>
```
Comment (wilfwilson) [inline doc/z-chap1.xml:112]: I think the flags would look better in `<C>` tags but whatevs 🙂 
```suggestion
<Item><C>--enable-code-coverage</C></Item><Item> enable code coverage support</Item>
```

File: README.md (hunk 1 above)
Comment (james-d-mitchell) [reply README.md:100]: Good point, I didn't think of that!

File: doc/z-chap1.xml (hunk 2 above)
Comment (james-d-mitchell) [reply doc/z-chap1.xml]: Agreed, looks swish in text mode :)

**Review by wilfwilson — APPROVED** [review]
Wonderful

File: README.md [hunk 3]
```diff
@@ -85,12 +85,27 @@ If you installed GAP on several architectures, then you must execute the
...
+
+In addition to the usual autoconf generated configuration flags, the following
+flags are provided.
+
+Option                        | Meaning
+----------------------------- | ------------------------------------------------
+--enable-code-coverage        | enable code coverage support
+--enable-compile-warnings     | enable compiler warnings
+--enable-debug                | enable debug mode
+--with-external-bliss         | use external bliss
```
Comment (alex-konovalov) [inline README.md:103]: maybe enclose `bliss` and `planarity` in backticks in this and the next line?

File: doc/z-chap1.xml [hunk 4]
```diff
@@ -94,6 +94,44 @@ make]]></Listing>
...
+   <Item><C>--enable-code-coverage</C></Item><Item> enable code coverage support</Item>
+ </Row>
+ <Row>
+   <Item><C>--enable-compile-warnings</C></Item><Item> enable compiler warnings</Item>
+ </Row>
+ <Row>
+   <Item><C>--enable-debug</C></Item><Item> enable debug mode</Item>
+ </Row>
+ <Row>
+   <Item><C>--with-external-bliss</C></Item><Item> use external bliss</Item>
```
Comment (alex-konovalov) [inline doc/z-chap1.xml:121]: Similarly, could use some markup here for the names of pieces of software.

File: README.md [hunk 5]
```diff
@@ -85,12 +85,27 @@ If you installed GAP on several architectures, then you must execute the
...
+--with-external-planarity     | use external planarity
```
Comment (wilfwilson) [inline README.md:104]:
```suggestion
--with-external-bliss         | use external `bliss`
--with-external-planarity     | use external `planarity`
```

File: doc/z-chap1.xml [hunk 6]
```diff
@@ -94,6 +94,44 @@ make]]></Listing>
...
+ </Row>
+ <Row>
+   <Item><C>--with-external-planarity</C></Item><Item> use external planarity</Item>
```
Comment (wilfwilson) [inline doc/z-chap1.xml:124]:
```suggestion
   <Item><C>--with-external-bliss</C></Item><Item> use external &bliss;</Item>
 </Row>
 <Row>
   <Item><C>--with-external-planarity</C></Item><Item> use external &edge-addition-planarity-suite;</Item>
```

**Review by wilfwilson — COMMENTED** [review]
I've added code suggestions to Alex's comments so that I can commit them now and get this PR merged.

**Pull request merged**